Models/card.cs

```diff
@@ -6,7 +6,16 @@
 
 namespace ATMEmulator.Models
 {
-    class Card
+    public class Card
     {
+        public string Individual { get; set; } //физическое лицо
+        public string BankName { get; set; }  //банк, который выпустил карту
+        public string CardType { get; set; } //тип карты (дебетовая или кредитная);
+        public int CreditLimit { get; set; } //кредитный лимит
+        public DateTime Validity { get; set; } //срок действия
+        public string Account { get; set; } //привязанный счёт
+        public bool NFC { get; set; } //поддержка бесконтактной оплаты
```

Comment:
Свойство, поле или переменную типа bool нужно называть в соответствии с правилами наименования:
> DO name Boolean properties with an affirmative phrase (CanSeek instead of CantSeek). Optionally, you can also prefix Boolean properties with "Is", "Can", or "Has", but only where it adds value.

Источник: [Names of Type Members
](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members)